components/layers/layers.tsx

```diff
@@ -0,0 +1,94 @@
+import {
```

Comment:
A lot of this is copied from [here](https://github.com/felt/js-sdk-starter-react/blob/main/src/components/LayersList.tsx) and adapted to our tech stack